lightning/src/ln/channelmanager.rs

```diff
@@ -3033,10 +3040,29 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 				}
 				true
 			});
+
+			channel_state.claimable_htlcs.retain(|&(ref payment_hash, _), htlcs| {
+				htlcs.retain(|htlc| {
+					if height >= htlc.cltv_expiry - CLTV_CLAIM_BUFFER - LATENCY_GRACE_PERIOD_BLOCKS {
```

Comment:
Or if I'm reading this wrong, maybe it would be clearer if `LATENCY_GRACE_PERIOD_BLOCKS` was moved to the other side of the inequality:

```rust
if height + LATENCY_GRACE_PERIOD_BLOCKS >= htlc.cltv_expiry - CLTV_CLAIM_BUFFER {
```

Reply:
`claimable_htlcs` isnt about peers, its about the user of RL finding the preimage and claim. Another way that may help would be to look at `LATENCY_GRACE_PERIOD_BLOCKS` as "the time it takes us to do a commitment update". I added a comment to this effect.

Reply:
Yep!